### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Docle**
Maybe, it should be 'dictionary' here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -734,7 +734,7 @@ What is going on here?
     2
     >>> dictionary in another_set
     True
-    >>> another_set.add(another_ordered_dict)
+    >>> another_set.add(dictionary)
```
Comment (satwikkansal) [inline README.md:737]: Actually, the [statement above it ](https://github.com/satwikkansal/wtfpython/pull/244/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R732)should be `dictionary`. Nice catch though, thanks..

**Comment by nifadyev** [comment]
Hey @Docle , are you planning to make requested changes and complete the PR?

**Comment by Docle** [comment]
> Hey @Docle , are you planning to make requested changes and complete the PR?

Sorry, I'd be happy to complete the PR, but I don't have any more git experience and don't know what to do next, can you help me complete it?

**Comment by nifadyev** [comment]
Sure, firstly, you'd better review @satwikkansal comment and make necessary changes. Then you should probably wait for #350 to be merged into main. Then you should setup a virtual environment, install dependencies, and run `nox` for changed snippet. Sorry, that these steps not in `CONTRIBUTION` guide, we are working on it at the moment.
Please don't hesitate to ask any occurred question.

**Comment by Docle** [comment]
> Sure, firstly, you'd better review @satwikkansal comment and make necessary changes. Then you should probably wait for #350 to be merged into main. Then you should setup a virtual environment, install dependencies, and run `nox` for changed snippet. Sorry, that these steps not in `CONTRIBUTION` guide, we are working on it at the moment. Please don't hesitate to ask any occurred question.

OH, after reviewing @satwikkansal comment this time, I think I misunderstood it before, there should be no problem here. I think I should close this request now. Sorry for the trouble.

**Pull request closed without merging**